### PR TITLE
DEP-2467: Configuration of Nginx. Configuring location access by cert…

### PR DESCRIPTION
**Pull request by carbone-kid**
…ificates issuers. Updated README.

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -21,6 +21,20 @@ For developing the buildpack, `pyenv` must be set up, and the dependencies for t
 
 Beware that an older file system is also supported which includes Python 3.4, so please take that into account while introducing additional dependencies or new dependency version.
 
+### Running in `Docker`
+As an alternative to running Python on your host you can run it in Docker container.
+* set up required environment variables
+* go to `./run-locally` folder
```
Comment (markbastiaans) [inline DEVELOPING.md:27]: I think we should check if the `run-locally` folder is the right place / name for the Dockerfile, composefile and bash script for the development environment. I would suggest putting these in a `dev` folder instead, but that's up for discussion.

File: DEVELOPING.md [hunk 2]
```diff
@@ -21,6 +21,20 @@ For developing the buildpack, `pyenv` must be set up, and the dependencies for t
...
+* execute `bash run-locally.sh`
+
+This will start the Docker container with preinstalled Python and "ssh" you into it.
+The project folder will be mapped to the current folder in the Docker container, so if you edit files on you host 
+the changes will be immediately available in the container.
+
+To run a test separately do:
+
+`nosetests --nocapture --verbosity=3 --processes=10 --process-timeout=3600 --with-timer --timer-no-color tests/integration/test-file-name.py`
```
Comment (markbastiaans) [inline DEVELOPING.md:36]: This might be better put in the `Testing` section of this document.

File: buildpack/nginx.py [hunk 3]
```diff
@@ -236,6 +238,18 @@ def get_path_config(samesite_cookie_workaround=False):
         if config.get("client-cert") or config.get("client_cert"):
             client_cert = "auth_request /client-cert-check-internal;"
 
+        issuer_dn = ""
```
Comment (markbastiaans) [inline buildpack/nginx.py:241]: Will this break if we initialise this as `None`? Python is quite different than Java in handling "empty" values.

File: DEVELOPING.md (hunk 1 above)
Comment (carbone-kid) [reply DEVELOPING.md]: Agree, dev is better

File: DEVELOPING.md (hunk 2 above)
Comment (carbone-kid) [reply DEVELOPING.md]: Moved to Testing

File: buildpack/nginx.py (hunk 3 above)
Comment (carbone-kid) [reply buildpack/nginx.py:226]: I would leave this for now.

**Pull request merged**